src/include/miopen/solver.hpp

```diff
@@ -2150,6 +2151,45 @@ struct ConvHipImplicitGemmWrwV4R4Xdlops_Padded_Gemm : SolverBase<ConvolutionCont
     Search(const ConvolutionContext&, const AnyInvokeParams& invoke_ctx) const;
 };
 
+struct PerformanceConvCkIgemmFwdV6r1DlopsNchw : Serializable<PerformanceConvCkIgemmFwdV6r1DlopsNchw>
+{
+    int ck_tunable_list_id;
```

Comment:
Having thought about how to implement it and having tried a version of implementation, this would make the function calls rather convoluted and hard to maintain.

I think if developer change the internal list of params (which is part of CK::solver), it means the MIOpen::solver has been changed because MIOpen::solver is calling into CK::solver, and therefore, MIOpen find-db and Perf-DB need to be updated.

Reply:
Ping

@JehandadKhan